core/plugins/org.polarsys.capella.core.ui.properties.richtext/src/org/polarsys/capella/core/ui/properties/richtext/sections/CapellaDescriptionPropertySection.java

```diff
@@ -63,10 +63,8 @@ public void dispose() {
       Set<CapellaDescriptionPropertySection> availableDescriptionSections = mapDescriptionSectionToEObject.keySet();
       for (CapellaDescriptionPropertySection descriptionSection : availableDescriptionSections) {
         if (descriptionSection != null && !descriptionSection.isDisplayedInWizard()) {
-          if (mapDescriptionSectionToEObject.get(descriptionSection) == element) {
-            descriptionSection.refresh();
-            descriptionSection.aboutToBeShown();
-          }
+          descriptionSection.refresh();
```

Comment:
I think it was done on purpose,
https://github.com/eclipse/capella/commit/c52365a209b226c4f3888e77aae32a841601f93f#diff-aef9be3be73ec713c464ab5ca75b80fcb446625a7d1c32b12fa7e77877ac8dd2R66-R69

can you check if it still reproductible on capella latest, as laurent has fixed something yesterday https://github.com/eclipse/capella/pull/2397

Reply:
after some test it works well, it's the other commit that need to be rollbacked
👍